runtime/development/src/lib.rs

```diff
@@ -771,7 +771,7 @@ impl pallet_crowdloan_claim::Config for Runtime {
 
 parameter_types! {
 	pub const LoanPalletId: PalletId = PalletId(*b"pal/loan");
-	pub const MaxLoansPerPool: u64 = 200;
+	pub const MaxLoansPerPool: u64 = 10;
```

Comment:
For my education, what triggers this change?

Reply:
This is actually mainly a change for testing, probably shouldn't be committed. The weight of the `updateNav` extrinsic is based on this constant. It turns out, the weight of updating the NAV of 200 loans is over the block limit, at least in the development runtime, hence I had to lower it to be able to call the extrinsic.